docs/provider-setup.md

```diff
@@ -85,9 +91,34 @@ domain = "http://192.168.56.102"
 #no_passphrase = true
 ```
 
-with suitable replacements.
+with suitable [replacements](#provider-options).
 
 Create the folders:
 ```(shell)
 curl http://192.168.56.102/cgi-bin/csaf_provider.go/create
 ```
+Or using the uploader:
+```(shell)
+./csaf_uploader -a create -u http://192.168.56.102/cgi-bin/csaf_provider.go
+```
+
+## Provider options
+Provider has many config options described as following:
+
+ - password: Authentication password for accessing the CSAF provider.
+ - key: The private OpenPGP key.
+ - folder: Specify the root folder. Default: `/var/www/`.
+ - web: Specify the web folder. Default: `/var/www/html`.
+ - tlps: Set the allowed TLP comming with the upload request (one or more of "csaf", "white", "amber", "green", "red").             
+   The "csaf" selection lets the provider takes the value from the CSAF document.         
+   These affects the list items in the web interface.      
+   Default: `["csaf", "white", "amber", "green", "red"]`.
+ - upload_signature: Send signature with the request, an additional input-field in the web interface will be shown to let user enter an ascii armored signature. Default: `false`.
+ - openpgp_url: URL to OpenPGP key-server. Default: `https://openpgp.circl.lu`.
+ - domain: The domain name. Default: `http://$SERVER_NAME`.
+ - no_passphrase: Let user send password with the request, if set to true the input-field in the web interface will be disappeared. Default: `false`.
+ - no_validation: Validate the uploaded CSAF document against the JSON schema. Default: `false`.
+ - no_web_ui: Disable the web interface. Default: `false`.
+ - dynamic_provider_metadata: Take the publisher from the CSAF document. Default: `false`.
+ - publisher: Set the publisher. Default: `{"category": "vendor", "name": "ACME", "namespace": "https:/example.com"}`.
```

Comment:
See comment about using ACME (ACME is a real company. Please use Example instead):
```suggestion
 - publisher: Set the publisher. Default: `{"category": "vendor", "name": "Example", "namespace": "https://example.com"}`.
```

Of course, that must be changed in the code as well

Reply:
Thanks for the question and the hint. We can use "Example" of course, though "ACME Corporation" seems to e a famous fictional company https://en.wikipedia.org/wiki/Acme_Corporation which is the ACME company that really exists (There was a car company, but spelled `Acme` ;) )